src/components/Quests/Quest2.js

```diff
@@ -4,28 +4,27 @@ import {
 	FormControl,
 	FormLabel,
 	Button,
-	Alert,
-	AlertIcon,
-	AlertTitle,
-	AlertDescription,
+	Box,
 } from '@chakra-ui/react'
 import React, { useState } from 'react'
 import useQuest from '../../hooks/useQuest'
 
-const SecondQuest = () => {
+const Quest2 = () => {
+	const [isSubmitted, setIsSubmitted] = useState(false)
 	const [formData, setFormData] = useState({
 		city: '',
 		school: '',
 		specialty: '',
 		interests: '',
 		hobby: '',
 	})
-
+	const questId = 1

```

Comment:
Same here, this should be the current Quest number - 2, but for the test we are pretending that this is Quest 1

Reply:
Got it ✅